pr_testing/test_multiple_prs.sh

```diff
@@ -1159,6 +1159,38 @@ for WF in ${WORKFLOWS_FOR_VALGRIND_TEST//,/ }; do
   popd
 done
 
+#
+# Enabled bot tests
+#
+for BT in ${ENABLE_BOT_TESTS}; do
+    if [ "$BT" = "PROFILING" ]; then
+        PROFILING_WORKFLOWS=$(echo $(grep "PR_TEST_MATRIX_EXTRAS_PROFILING=" $CMS_BOT_DIR/cmssw-pr-test-config | sed 's|.*=||'), | tr ' ' ','| tr ',' '\n' | grep '^[0-9]' | sort | uniq | tr '\n' ',' | sed 's|,*$||')
+         pushd $WORKSPACE
+         git clone https://github.com/cms-cmpwg/profiling.git
+         popd
+         mark_commit_status_all_prs 'profiling' 'pending' -u "${BUILD_URL}" -d "Running tests" || true
+         report_pull_request_results_all_prs_with_commit "TESTS_RUNNING" --report-pr ${REPORT_H_CODE} --pr-job-id ${BUILD_NUMBER} --add-message "Running Profling" ${NO_POST}
+         for PROFILING_WORKFLOW in $PROFILING_WORKFLOWS;do
+             $WORKSPACE/profiling/Gen_tool/Gen.sh $CMSSW_IB
+             $WORKSPACE/profiling/Gen_tool/runall.sh $CMSSW_IB
+             $WORKSPACE/profiling/Gen_tool/runall_cpu.sh $CMSSW_IB
+             $WORKSPACE/profiling/Gen_tool/runall_mem.sh $CMSSW_IB
+             pushd $WORKSPACE/$CMSSW_IB/src/$PROFILING_WORKFLOW
+             ./profile.sh $CMSSW_IB
+             popd
```

Comment:
protect all the above commands so that if they fail then it should not break the jenkins job. We do want to continue and report at the end if PROFILING was fail. So make sure if PROFILING is enabled then this code should not break the jenkins job

Reply:
On the cmsbuild machine it took almos 8 hours
https://cmssdt.cern.ch/jenkins/job/test-pr-run-tests/13/

Reply:
If I can run a 4 thread job  on lxplus I can move it there.

Reply:
PR tests will run on cmsbuild nodes. Currently for PR tests every thing needs to be build and run on the same node.